Public/Src/Sandbox/MacOs/Sandbox/Sandbox.xcodeproj/xcshareddata/xcschemes/SandboxMonitor.xcscheme

```diff
@@ -17,13 +17,13 @@
                BlueprintIdentifier = "F51A2BF82190C67500880752"
                BuildableName = "SandboxMonitor"
                BlueprintName = "SandboxMonitor"
-               ReferencedContainer = "container:BuildXLSandbox.xcodeproj">
+               ReferencedContainer = "container:Sandbox.xcodeproj">
```

Comment:
>Sandbox [](start = 48, length = 7)

intentional? Didn't see the xcode project renamed...

Reply:
The scheme was referencing both, the removed non-existent one and the right one (Sandbox.xcodeproj). The latter was already present and the former just a dead reference.